examples/server/server.cpp

```diff
@@ -735,14 +738,17 @@ struct server_task_result_rerank : server_task_result {
     int index = 0;
     float score = -1e6;
 
+    int32_t n_prompt_tokens;
```

Comment:
This seems like a better name in this context:

```suggestion
    int32_t n_tokens;
```

Same for `server_task_result_embd`.

Reply:
Right, `n_tokens` indeed makes more sense here.  Fixed.